keggtools/resolver.py

```diff
@@ -54,18 +56,19 @@ def get_gene_names(genes: List[str], max_genes: int = 50) -> Dict[str, str]:
     #         )
 
     # Build query string
-    query_string: str = "+".join(genes)

```

Comment:
Hi. The plus is required here. It indicates the concatination of multiple gene identifier.